packages/nx/src/command-line/release/version/release-group-processor.ts

```diff
@@ -736,6 +759,59 @@ export class ReleaseGroupProcessor {
     };
   }
 
+  async processDockerProjects(dockerVersionScheme?: string) {
+    const dockerProjects = new Map<string, FinalConfigForProject>();
+    for (const project of this.versionData.keys()) {
+      const finalConfigForProject = this.finalConfigsByProject.get(project);
+      if (Object.keys(finalConfigForProject.dockerOptions).length === 0) {
+        continue;
+      }
+      dockerProjects.set(project, finalConfigForProject);
+    }
+    // If no docker projects to process, exit early to avoid unnecessary loading of docker handling
+    if (dockerProjects.size === 0) {
+      return;
+    }
+    let handleDockerVersion: (
+      workspaceRoot: string,
+      projectGraphNode: ProjectGraphProjectNode,
+      finalConfigForProject: FinalConfigForProject,
+      dockerVersionScheme?: string
+    ) => Promise<{ newVersion: string; logs: string[] }>;
+    try {
+      const {
+        handleDockerVersion: _handleDockerVersion,
+        // nx-ignore-next-line
+      } = require(// @ts-ignore
+      '@nx/docker/release/version-utils');
+      handleDockerVersion = _handleDockerVersion;
+    } catch (e) {
+      console.error(
+        'Could not find `@nx/docker`. Please run `nx add @nx/docker` before attempting to releas Docker images.'
```

Comment:
nit: There's a typo here 
```suggestion
        'Could not find `@nx/docker`. Please run `nx add @nx/docker` before attempting to release Docker images.'
```